cv_next/components/layout/navbar/mode-toggle.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
```

Comment:
OK I know you probably didn't write this component as you've only switched dirs here, but pls do the following:
a. change the file name to match the standard component naming (I think this file, along with others, are from some example we took a while back and didn't handle correctly; pls see other component files that are with snake-case instead of camelCase).
b. don't `import * as React`, import the specifics needed.
c. we have a `isBrowser` util in the lib folder, use it instead of the `isClient` state.

Reply:
Changed all notes
Also check for other import * and fixed these